recipes/py_stringmatching/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.4.0" %}
+{% set name = "py_stringmatching" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 64794c179cefece8f337c9f458dfdce937b08efabad220e2dcdad81be3e95a66
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - setuptools
+    - numpy >=1.7.0
```

Comment:
Please see the following about building packages that link against `numpy`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-against-numpy